code/modules/paperwork/pen.dm

```diff
@@ -68,25 +68,22 @@
 /obj/item/weapon/pen/fountain
 	name = "fountain pen"
 	desc = "It's a common fountain pen, with a faux wood body."
-	icon_state = "pen-fountain"
+	icon_state = "fountainpen"
 	font = FOUNTAIN_PEN_FONT
 
 /obj/item/weapon/pen/fountain/captain
 	name = "captain's fountain pen"
 	desc = "It's an expensive Oak fountain pen. The nib is quite sharp."
-	icon_state = "pen-fountain-o"
+	icon_state = "fountainpen"
 	force = 5
 	throwforce = 5
 	throw_speed = 4
 	colour = "crimson"
 	materials = list(MAT_GOLD = 750)
 	sharpness = IS_SHARP
 	resistance_flags = FIRE_PROOF
-	unique_reskin = list("Oak" = "pen-fountain-o",
-						"Gold" = "pen-fountain-g",
-						"Rosewood" = "pen-fountain-r",
-						"Black and Silver" = "pen-fountain-b",
-						"Command Blue" = "pen-fountain-cb"
+	unique_reskin = list("Gold" = "goldpen",
+						"Black and Silver" = "fancypen",

```

Comment:
No sprites for them, so, instead of doing what the bartender shotgun does, that it leaves the missing sprite loadout there so someone chooses it and it goes invisible, i decided to remove the spriteless ones